CHANGELOG.md

```diff
@@ -7,6 +7,58 @@
+- Drop Crosswalk library that has been unsupported since 2016.
+- Better support for upcoming versions of CHT Core.
+- Smaller APK size. Before it was 746KB, now it is 381KB.
+
+### Upgrade steps
+
+Install CHT Android v1.0.0 in the phone, then upgrade [WebView](https://play.google.com/store/apps/details?id=com.google.android.webview) to the latest available version in Google Play Store and enable automatic updates of this app by following the [Google Documentation](https://support.google.com/googleplay/answer/113412). This will ensure top performance when using CHT.
+
+### Performance improvements
+
+This is the performance test result that compares CHT Android v0.8.0 with the new CHT Android v1.0.0.
+- **Device:** Sony Xperia Z1
+- **Android:** 5.1.1
+- **CHT Core:** 3.15.0
+
+| Test | v0.8.0 | v1.0.0 | Improvement |
+| ------------ | ------------ | ------------ | ------------ |
+| Initial load (fetching 1882 documents) | 0:03:14  | 0:01:35  | **204%** |
+| Refreshing after initial replication | 0:00:10  | 0:00:03  | **333%** |
+| Loading Contacts tab | 0:00:17  |  0:00:12 | **142%** |
+| Loading Reports tab | 0:00:23  | 0:00:21  | **109%** |
+| Loading a specific report (New Pregnancy) | 0:00:04  |  0:00:02 | **200%** |
+| Track opening a form (New Pregnancy) | 0:00:05  | 0:00:04  | **125%** |
+| Loading Tasks tab | 0:00:02  | 0:00:01  | **200%** |
+
+### Changes
+
+- [improvement] Check and handle when location permission is denied or granted [#189](https://github.com/medic/cht-android/issues/189).
+- [improvement] Request "Approximate" and "Precise" location since they are required in Android 12 [#207](https://github.com/medic/cht-android/issues/207).
+- [improvement] Replace deprecated ImagePicker [#159](https://github.com/medic/cht-android/issues/159).
+- [performance] Android app should navigate to root instead of the _rewrite path after webapp v3.5.x [#94](https://github.com/medic/cht-android/issues/94).
+- [feature] Make it clear when a training app is in use [#258](https://github.com/medic/cht-android/issues/258).
+- [bug] Border not showing red for production URL in dev app [#266](https://github.com/medic/cht-android/issues/266).
+- [bug] Prominent disclosure when "handling users' Files" [#148](https://github.com/medic/cht-android/issues/148).
+- [deprecation] Remove unbranded-test flavor [#257](https://github.com/medic/cht-android/issues/257).
+- [deprecation] Remove the old menu and lock pin screen [#253](https://github.com/medic/cht-android/issues/253).
+
+### Development changes
+
+- [improvement] Upgrade to Android 12 [#205](https://github.com/medic/cht-android/issues/205).
+- [improvement] Migrate off of the unmaintained com.noveogroup.android:check plugin [#242](https://github.com/medic/cht-android/issues/242).
+- [improvement] PMD Processing errors for files that are not .java [#255](https://github.com/medic/cht-android/issues/255).
+
+### Breaking changes
```

Comment:
Awesome. Let's move this section to above `### Changes` because it's very important that people understand this before anything else.